### ISSUE TITLE
Compilation not working with Java 11

### ISSUE THREAD

**Issue by berquist** [NextCenturyCorporation/AIDA-Interchange-Format#113]
It looks like a problem with my Java install, but I get a similar error on an Arch Linux machine that goes away when switching to Java 8. I've also tried `git clean -xdf .` and deleting `~/.m2`.

Doing `mvn compile`:
```
[INFO] Scanning for projects...
[INFO]
[INFO] ----------------------< com.ncc:aida-interchange >----------------------
[INFO] Building aida-interchange 1.0-SNAPSHOT
[INFO] --------------------------------[ jar ]---------------------------------
[INFO]
[INFO] --- maven-resources-plugin:2.6:resources (default-resources) @ aida-interchange ---
[WARNING] Using platform encoding (UTF-8 actually) to copy filtered resources, i.e. build is platform dependent!
[INFO] Copying 15 resources
[INFO]
[INFO] --- maven-compiler-plugin:3.3:compile (default-compile) @ aida-interchange ---
[WARNING] File encoding has not been set, using platform encoding UTF-8, i.e. build is platform dependent!
[INFO] Compiling 6 source files to /Users/berquist/projects/aida/aida_interchange_format/repos/AIDA-Interchange-Format/target/classes
/Users/berquist/projects/aida/aida_interchange_format/repos/AIDA-Interchange-Format/src/main/java/com/ncc/aif/SeedlingOntologyMapper.java:8: error: cannot access HashSet
import java.util.HashSet;
                ^
  bad class file: /modules/java.base/java/util/HashSet.class
    class file has wrong version 55.0, should be 53.0
    Please remove or make sure it appears in the correct subdirectory of the classpath.
1 error
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  1.414 s
[INFO] Finished at: 2019-01-23T17:15:55-05:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.3:compile (default-compile) on project aida-interchange: Compilation failure -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
```
The environment:
```
Apache Maven 3.6.0 (97c98ec64a1fdfee7767ce5ffb20918da4f719f3; 2018-10-24T14:41:47-04:00)
Maven home: /usr/local/Cellar/maven/3.6.0/libexec
Java version: 11.0.2, vendor: Oracle Corporation, runtime: /Library/Java/JavaVirtualMachines/openjdk-11.0.2.jdk/Contents/Home
Default locale: en_US, platform encoding: UTF-8
OS name: "mac os x", version: "10.14.2", arch: "x86_64", family: "mac"
```

**Comment by nextcen-dgemoets** [comment]
Currently we've only built and tested under Java 8.  Did AIF used to build for you under Java 11 before the most recent updates (that is, before the addition of the Java AIF in the com.ncc.aif package)?

I've written up a Task in our issue tracker to migrate AIF to a later version of Java, and have cited this issue.

**Comment by berquist** [comment]
Actually, I don't think so. But it's a non-issue for me personally at this point since Java 8 is working fine.

**Comment by nextcen-dgemoets** [comment]
Fixed in [AIF v1.1.1](https://github.com/NextCenturyCorporation/AIDA-Interchange-Format/releases/tag/tag%2Fv1.1.0).  Closing.
